diff --git a/node_modules/@measured/puck/dist/resolve-all-data-ppm982Jh.d.ts b/node_modules/@measured/puck/dist/resolve-all-data-ppm982Jh.d.ts
index dd7f77f..e9955eb 100644
--- a/node_modules/@measured/puck/dist/resolve-all-data-ppm982Jh.d.ts
+++ b/node_modules/@measured/puck/dist/resolve-all-data-ppm982Jh.d.ts
@@ -22,7 +22,8 @@ type FieldOption = {
 type FieldOptions = Array<FieldOption> | ReadonlyArray<FieldOption>;
 type BaseField = {
     label?: string;
-};
+    validator?: (value: any) => { message: string; valid: boolean }
+}
 type TextField = BaseField & {
     type: "text";
 };
